src/utils/sortByRelevance.tsx

```diff
@@ -7,8 +7,12 @@ import { sortBy } from "lodash";
  * @param target The target string to compare against
  * @returns A new array of sorted strings
  */
-export function sortByRelevance(items: string[], target: string): string[] {
-  return sortBy(items, [
+export function sortByRelevance(
```

Comment:
please also rename this file and make it a common places for future helper methods, **OR** move this method to the existing helpers file if there is one already. This kind of method shouldn't be a standalone component.

Reply:
right now, it is only 1 - seems no problems at all, but you will have a mess in future if keep creating new standalone components this way unnecessarily, hard to read, hard to maintain, hard to debug.

Reply:
Thanks Viets. I have refactored most of the utils folder, but didn't move files like AppRouter.tsx because for now hard to find a place to put this. I will do this in other ticket for updating project folder/file structure.

Reply:
there are `TODO` added to them, should be enough for this scope.